src/Features/CSharp/Portable/MakeMethodAsynchronous/CSharpMakeMethodAsynchronousCodeFixProvider.cs

```diff
@@ -35,7 +35,9 @@ protected override string GetMakeAsyncVoidFunctionResource()
 
         protected override bool IsMethodOrAnonymousFunction(SyntaxNode node)
         {
-            return node.IsKind(SyntaxKind.MethodDeclaration) || node.IsAnyLambdaOrAnonymousMethod();
+            return node.IsKind(SyntaxKind.MethodDeclaration)
+                || node.IsAnyLambdaOrAnonymousMethod()
+                || node.IsKind(SyntaxKind.LocalFunctionStatement);
```

Comment:
I couldn't find an existing extension for this. I always wasn't sure what to call it or where to put it if we did have it. For now I left it but if you have a more specific suggestion I can implement that. The functionality does seem specific to just these two operations, at least for now.